src/cascadia/TerminalSettingsModel/FontConfig.cpp

```diff
@@ -30,6 +30,17 @@ winrt::com_ptr<FontConfig> FontConfig::CopyFontInfo(const FontConfig* source, wi
     MTSM_FONT_SETTINGS(FONT_SETTINGS_COPY)
 #undef FONT_SETTINGS_COPY
 
+    // We cannot simply copy the font axes and features with `fontInfo->_FontAxes = source->_FontAxes;`
+    // since that'll just create a reference; we have to manually copy the values.
+    if (source->_FontAxes)
+    {
+        fontInfo->_FontAxes = winrt::single_threaded_map<winrt::hstring, float>();
+        for (const auto keyValuePair : source->_FontAxes.value())
+        {
+            fontInfo->_FontAxes.value().Insert(keyValuePair.Key(), keyValuePair.Value());

```

Comment:
Try to avoid WinRT container classes whenever possible. They have a very high overhead (both in performance and binary size). Here for instance, we can avoid it by building a vector first and only then converting it to a WinRT container. Like so (pseudocode):
```cpp
std::map<winrt::hstring, float> axes;
for (const auto keyValuePair : source->_FontAxes.value())
{
    axes.insert(keyValuePair.Key(), keyValuePair.Value());
}
fontInfo->_FontAxes = winrt::single_threaded_map(std::move(axes));
```